odbc/2.linux.md

```diff
@@ -0,0 +1,92 @@
+# Linux ODBC Setup
+
+## Driver Manager Installation
+
+On Linux, we will be using unixODBC as our driver manager. To install, use the package manager that comes with your system and search for `unixODBC*`, which should return all packages available to you that begin with "unixODBC". You will have to download `unixODBC` and `unixODBC-devel` or similar (may be named different things on different Linux distributions).
+
+For RedHat, a Linux distribution that uses `yum` as its packag manager (much like IBM i), the following command and output shows how to check for the correct packages to download:
+
+**RedHat example**
+```bash
+$ yum list available unixODBC*
+Available Packages
+unixODBC.i686                                2.3.1-11.el7                             RHEL-76-x86_64
+unixODBC-devel.i686                          2.3.1-11.el7                             RHEL-76-x86_64
+```
+
+Once you know what packages are needed, they can be downloaded also through your package manager:
+
+**RedHat example**
+```bash
+$ yum install unixODBC unixODBC-devel
+```
+
+You now have unixODBC installed on your system
```

Comment:
There is no need to install unixODBC separately, since the driver will pull it in automatically when you install it.